README.md

```diff
@@ -29,7 +29,7 @@ toStream('README.md')
   .pipe(stats())
   .on('data', function (file) {
     // adds `stat` object to file
-    console.log(file.stat);
+    console.log(file.stat.toString());
   })
```

Comment:
sorry if I'm forgetting something... this is just an object here right? so should that be `console.log(JSON.stringify(file.stat))` or just `console.log(file.stat)`?